pepper-apis/dsm-core/src/test/java/org/broadinstitute/dsm/util/TestParticipantUtil.java

```diff
@@ -79,4 +95,17 @@ public static ParticipantDto createIneligibleSharedLearningParticipant(String pa
                 "elastic/participantProfile.json", "elastic/participantDsm.json",
                 "elastic/lmsActivitiesSharedLearningIneligible.json");
     }
+
+    /**
+     * Create a profile with a unique HRUID (GUID assignment left to caller), and email constructed from first and
+     * last name.
+     */
+    public static Profile createProfile(String firstName, String lastName, int uniqueDigits) {
+        Profile profile = new Profile();
+        profile.setEmail(String.format("%s%s@broad.org", firstName, lastName));
```

Comment:
If there's any chance of emails being sent somewhere as a result of this, let's use `@broad.dev` since `@broad.org` goes somewhere else.